Contents.swift

```diff
@@ -102,36 +112,48 @@ greaterNumbers(array: [110, 20, 100, 0, 200], number: 100)
 //
 //printIfPositiveInteger(number: "10")
 
-//2 guard statements
-func printIfPositiveInteger(number: String) {
-
-    guard let value = Int(number) else {
-        return
-    }
-
-    guard value > 0 else {return}
+class PositiveInteger {
+    //2 guard statements
+    func printIfPositiveInteger(number: String) -> Int {
```

Comment:
```suggestion
    func printIfPositiveInteger(number: String) -> Int {
```

it-s not printing anything

Reply:
done